packages/next/migrations.json

```diff
@@ -119,6 +125,19 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "12.6.0": {
+      "version": "12.6.0-beta.0",
+      "packages": {
+        "next": {
+          "version": "11.0.1",
+          "alwaysAddToPackageJson": false
+        },
+        "eslint-config-next": {
```

Comment:
As discussed, let's move it to the actual migration so we can install it as a dev dependency.

Reply:
i'm hoping that https://github.com/nrwl/nx/pull/6287 will be merged soon, and then i can just use that, tbh. but if that PR will take too long, i'll continue with this PR without it

Reply:
updated PR to not rely on #6287 anymore, so we can merge this PR sooner rather than later